.cspell.json

```diff
@@ -0,0 +1,258 @@
+        "doloe",
+        "amet",
+        "interdum",
+        "molestie",
+        "pretium",
+        "lacus",
+        "sagittis",
+        "laoreet",
+        "Cpath",
+        "Csvg",
+        "slottedsvg",
+        "viewbox",
+        "facilisis",
+        "Aliquam",
+        "dignissim",
+        "arcu",
+        "egestas",
+        "tempor",
+        "sodales",
+        "quis",
+        "maxage",
+        "architechtural",
+        "necolas",
+        "keyline",
+        "visiblity",
+        "slotchange",
+        "canvastext",
+        "appload",
+        "vname",
+        "vkey",
+        "vchildren",
+        "vattrs",
+        "vtext",
+        "vtag",
+        "autocorrection",
+        "mollis",
+        "faucibus",
+        "consequat",
+        "Curabitur",
+        "commodo",
+        "pharetra",
+        "volutpat",
+        "luctus",
+        "felis",
+        "Donec",
+        "elementum",
+        "Cras",
+        "dereq",
+        "memoizerific",
+        "unbinders",
+        "testid",
+        "Memoizerific",
+        "Applicationwithareallyreallylongsinglewordname",
+        "Dont",
+        "Liberica",
+        "Robusta",
+        "Arabica",
+        "Nonenumerable",
+        "ceeb",
+        "aliquip",
+        "laboris",
+        "ullamco",
+        "nostrud",
+        "veniam",
+        "justo",
+        "accumsan",
+        "rhoncus",
+        "sollicitudin",
+        "himenaeos",
+        "sociosqu",
+        "vulputate",
+        "fullheight",
+        "focussable",
+        "Cancelation",
+        "interactable",
+        "grouplinks",
+        "elit",
+        "eiusmod",
+        "incididunt",
+        "fakepath",
+        "Nunito",
+        "fontsource",
+        "WHCM",
+        "dismissable",
+        "uncollapsing",
+        "interactiveelements",
+        "svgs",
+        "searchbar",
+        "testbutton",
+        "hendrerit",
+        "aliquet",
+        "Antidisestablishmentarianismbutmadeevenlonger",
+        "Reallylongonewordanswerthatcouldbethenameofacoffee",
+        "Praesent",
+        "nibh",
+        "screenreaders",
+        "tidyup",
+        "horizonatally",
+        "euismod",
+        "tincidunt",
+        "porttitor",
+        "mazagran",
+        "frappuccino",
+        "affogato",
+        "panna",
+        "luwak",
+        "kopi",
+        "galão",
+        "placerat",
+        "feugiat",
+        "augue",
+        "ultricies",
+        "eleifend",
+        "nonummy",
+        "myfunc",
+        "visuallyhidden",
+        "someurl",
+        "undefinedpx",
+        "Nrem",
+        "typograpghy",
+        "lobortis",
+        "merker",
+        "trunc",
+        "Gatbsy",
+        "mycontolname",
+        "imperdiet",
+        "autoprops",
+        "hiden",
+        "scrolltotop",
+        "texy",
+        "icoptionselect",
+        "searchoption",
+        "validationinline",
+        "hidel",
+        "prepopulate",
+        "texfield",
+        "shold",
+        "destuctive"
```

Comment:
Some of these words should be removed: e.g. "destuctive", "typograpghy", "GHSA" etc.